cost-analyzer/values.yaml

```diff
@@ -199,6 +199,7 @@ oidc:
   authURL: "https://my.auth.server/authorize" # endpoint for login to auth server
   loginRedirectURL: "http://my.kubecost.url/model/oidc/authorize" # Kubecost url configured in provider for redirect after authentication
   discoveryURL: "https://my.auth.server/.well-known/openid-configuration" # url for OIDC endpoint discovery
+  hostedDomain: "domain" # blocks access to specified domain only
```

Comment:
Needs better default example. Is this talking about the auth domain or the application/service domain?

Reply:
thank you for your feedback, @LaikaN57! I've updated the description to capture more detail - if set, we expect the value of `hostedDomain` to match the `hd` claim part of the ID token supplied during the authentication process.